src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/haflow/create/actions/ResourcesAllocationAction.java

```diff
@@ -213,7 +213,7 @@ private void allocateMainPath(HaFlowCreateFsm stateMachine) throws UnroutableFlo
 
             HaFlowPath forward = flowPathBuilder.buildHaFlowPath(
                     haFlow, haFlowResources.getForward(), paths.getForward(),
-                    cookieBuilder.direction(FlowPathDirection.FORWARD).subType(FlowSubType.SHARED).build());
+                    cookieBuilder.direction(FlowPathDirection.FORWARD).subType(FlowSubType.SHARED).build(), false);
```

Comment:
It is always called with `false` because it is just preparation for reroute PR #5204 
I just want to split big PR into 2 PRs and moved moved into current PR as much code as I can.
What do you mean by "descriptive method"?

Reply:
like `buildHaFlowPathWithForseIngnore` and `buildHaFlowPathWithoutForceIgnore`? 
From my point of view it is less helpful, because
1. You have to create 3 methods with almost same signature
2. You have to create a java doc for each of them, which will describe almost same set of arguments
3. Idea helps you to understand what `false` means on the caller side by argument hints 
![image](https://github.com/telstra/open-kilda/assets/8777565/c7e914c3-5534-4a78-9ef3-92f20091121d)

Reply:
but not in the review. If there is a typo, or a copy-paste issue, it is unlikely that somebody on review will spot it. You don't need the third signature, because you only invoke it with `false`.
An alternative could be to use a constant describing the literal.

Reply:
Frankly, I don't get your point. This is a problematic point in our code base: (over-) using method signatures with a lot of arguments (specifically with booleans) and call these methods with literals. This leads to difficult-to-read code. A person needs to remember these signatures or to use tools that would remind what those signatures do (like idea). In my experience, people rarely do these things; in reality they simply skip these parts in the review process. And this is how a bug put a leg into the door. 
There are ways to mitigate this problem: descriptive method names, descriptive constant names, encapsulate it into another object, pull that logic up, and others. Yes, there is another alternative: continue to use it as it is now. But then we continue having the problem I described. So my message here is: when weighing pros and cons of any of the approaches let's include the difficulty of review thing. 
Specifically here, I cannot easily understand the meaning of many true and false, and I cannot tell whether there is a mistake/typo in any of them. And don't think others can do. As for massive, we look at the impl side once but we look at the usage part many times. So that "massiveness" thing is not applicable to all of the places except one. When I weight `it's massive in one class, but it makes it clear in 10 other places` or `it's shorter in one class, but it makes it harder to understand in other 10 places` I choose the first one.

Reply:
Okay. Regarding to "An alternative could be to use a constant describing the literal."
Do you mean `private static final DON_IGNORE_BANDWIDTH = false` or just local `boolean forceIgnoreBandwidth = false`?